app/src/main/java/pl/droidevs/books/library/BookActivity.java

```diff
@@ -0,0 +1,115 @@
+package pl.droidevs.books.library;
+
+import android.app.Activity;
+import android.content.Loader;
+import android.content.res.Resources;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.os.AsyncTask;
+
+import android.support.v4.content.ContextCompat;
+import android.support.v4.content.res.ResourcesCompat;
+import android.support.v4.graphics.ColorUtils;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.graphics.Palette;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+
+
+import net.opacapp.multilinecollapsingtoolbar.CollapsingToolbarLayout;
+
+import java.security.PKCS12Attribute;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import pl.droidevs.books.R;
+
+public class BookActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        getSupportActionBar().hide();
+        setContentView(R.layout.activity_book);
+
+        ImageView imageView = (ImageView) findViewById(R.id.albumImageView);
+        Glide.with(this).load(R.drawable.orient_express).into(imageView);
```

Comment:
Nie musisz używać Glida, aby pobierać zasoby, które są wbudowane w aplikację. To jest tylko do testu 😉 ?

Reply:
Obrazek tak. Ale Glide zamierzam cały czas wykorzystywać ;)